src/providers/walletconnectNotify.ts

```diff
@@ -86,12 +95,12 @@ function formatMessage(event: Event, proposal) {
   if (!space) return null;
 
   const notificationType = WALLETCONNECT_NOTIFICATION_TYPE;
-  const notificationBody = `🟢 New proposal on ${space.name} @${space.id}\n\n`;
+  // const notificationBody = `🟢 New proposal on ${space.name} @${space.id}\n\n`;
 
   const url = `${proposal.link}?app=web3inbox`;
   return {
-    title: proposal.title,
-    body: notificationBody,
+    title: truncate(proposal.title, 64),
+    // body: notificationBody,
```

Comment:
We need body too no? 

it look like this before for example 
<img width="636" alt="Untitled 3" src="https://github.com/snapshot-labs/snapshot-webhook/assets/15967809/4d9e587a-1efd-4f88-a52a-715074fcb2d0">

Reply:
Correct